master-nonlatent/master.cfg

```diff
@@ -937,7 +937,7 @@ f_macos.addStep(
 f_macos.addStep(
     steps.ShellCommand(
         name="cleanup",
-        command="rm -r * .*i ~/packages/* ~/buildbot_logs/* 2> /dev/null || true",
+        command="rm -r * .*i ~/packages/* ~/buildbot_logs/* /System/Volumes/Data/cores/* 2> /dev/null || true",
```

Comment:
This looks a bit dangerous to me (maybe absolutely not). Can't you make sure that you are in the correct directory?
I am talking about `rm -r *` ?

Reply:
MTR creates some symlinks to /cores and apparently are stored there

`ec2-user@ip-172-31-26-17 mysql-test % ls -la ./var/4/log/binlog_encryption.rpl_typeconv-row/core.3990
lrwxr-xr-x  1 ec2-user  staff  16 Nov 22 13:19 ./var/4/log/binlog_encryption.rpl_typeconv-row/core.3990 -> /cores/core.3990

pare ca /cores se duce undeva dubios pe mac`